manifests/s/Swift/Toolchain/5.6.1/Swift.Toolchain.locale.en-US.yaml

```diff
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=AUSU.5-1-22567-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Swift.Toolchain
+PackageVersion: "5.6.1"
+PackageLocale: en-US
+Publisher: dt.compnerd.org
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+Author: Swift Open Source Project
+PackageName: Swift
+PackageUrl: https://www.swift.org/
+License: Copyright (c) Swift Open Source Project
```

Comment:
A copyright is not a license; Can you please update this to the actual license terms such as `Proprietary`, `MIT`, `GPL-2.0`, or whatever license applies

Reply:
I'm not sure how this has managed to stay there for so long.  Thank you for noticing the issue!